user/languages/android.md

```diff
@@ -200,6 +200,52 @@ As for any JVM language, it is also possible to [test against multiple JDKs](/us
 
 For Android projects, `env` and `jdk` can be given as arrays to construct a build matrix.
 
+## Building Android projects on new build environments
+
+The `dist: trusty` build environment is the only supported build environment for Android but if you would like to build on newer build environments e.g. `dist: jammy`, you can exercise your access to the Travis CI build environments and install required packages and tools. An example .travis.yml config can be reviewed below:
+
+```yaml
+os: linux
+language: java
+jdk: openjdk17
+
+env:
+ global:
+  - ANDROID_HOME=$HOME/travis-tools/android
+  - ANDROID_SDK_ROOT=$HOME/travis-tools/android
+
+before_install:
+ # PREPARE FOR ANDROID SDK SETUP
+ - mkdir -p $HOME/travis-tools/android && mkdir $HOME/.android && touch $HOME/.android/repositories.cfg
+ - cd $ANDROID_HOME && wget -q "https://dl.google.com/android/repository/commandlinetools-linux-10406996_latest.zip" -O commandlinetools.zip
+ - unzip -q commandlinetools.zip && cd cmdline-tools
+ - mv * tools | mkdir tools && cd $TRAVIS_BUILD_DIR
+ 
+ # SETUP PATH(s)
+ - export PATH=$ANDROID_HOME/cmdline-tools/tools/bin/:$PATH
+ - export PATH=$ANDROID_HOME/emulator/:$PATH
+ - export PATH=$ANDROID_HOME/platform-tools/:$PATH
+
+install:
+ # INSTALL REQUIRED ANDROID SDK TOOLS
+ - sdkmanager --sdk_root=$ANDROID_HOME --list | awk '/Installed/{flag=1; next} /Available/{flag=0} flag'
+ - yes | sdkmanager --sdk_root=$ANDROID_HOME --install "platform-tools" "platforms;android-33" "build-tools;33.0.2" "emulator" "system-images;android-33;google_apis;x86_64"
+ - sdkmanager --list --sdk_root=$ANDROID_HOME | awk '/Installed/{flag=1; next} /Available/{flag=0} flag'
+ # CREATE AVD
+ - echo "no" | avdmanager --verbose create avd --force --name "my_android_33" --package "system-images;android-33;google_apis;x86_64" --tag "google_apis" --abi "x86_64"
+ - sudo chmod -R 777 /dev/kvm
+ # Start emulator in background and wait for it to start
+ - adb kill-server && adb start-server &
+ - sleep 15
```

Comment:
Please add a comment in this line like

`  - sleep 15 # forceful sleep value may need to be adjusted depending on the specific build environment`